t/03-testapi.t

```diff
@@ -421,7 +440,7 @@ subtest 'check_assert_screen' => sub {
         is_deeply($autotest::current_test->{details}, [
                 {
                     result => 'unk',
-                    screenshot => 'basetest-13.png',
+                    screenshot => 'basetest-14.png',
```

Comment:
How is this different now? I don't see any changes in assets or jobs 🤔

Reply:
It is different because the newly added test inserts a test detail so subsequent test details will have a higher number.

Reply:
Ah. I totally missed that it writes the result to disk, too. Makes sense!